src/invidious/views/components/search_box.ecr

```diff
@@ -0,0 +1,9 @@
+<form class="pure-form" action="/search" method="get">
+	<fieldset>
+		<input type="search" id="searchbox" autocomplete="off" autocorrect="off"
+		autocapitalize="none" spellcheck="false" autofocus="" name="q"
```

Comment:
Looked up mdn docs for autofocus and it says it's a boolean attribute. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

Any reason for setting it to `=""`?

Reply:
Ah, it's because I've seen it being used like this. Probably on some random help site. I'll fix that.